java/piranha/src/test/java/com/uber/piranha/CorePiranhaTest.java

```diff
@@ -1134,6 +1134,29 @@ public void testMethodTestDoNotAllowsChainFlag() throws IOException {
         .doTest();
   }
 
+  @Test
+  public void testMethodTestDoNotAllowsMatchingArgMethodInvc() throws IOException {
```

Comment:
the counterpart test for this? Where matching is allowed?

Reply:
It is tested as part of the other tests for control and treated.